<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManagerWidget</class>
 <widget class="QWidget" name="ManagerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="userCheckTab">
       <attribute name="title">
        <string>유저 조회</string>
       </attribute>
       <widget class="QTableWidget" name="userSearchTableWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>377</width>
          <height>196</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>사용자 아이디</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>계좌 종류</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>잔액</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>기간</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>계좌 생성일</string>
         </property>
        </column>
       </widget>
       <widget class="QLineEdit" name="userSearchlineEdit">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>0</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="userDeleteTab">
       <attribute name="title">
        <string>유저 삭제</string>
       </attribute>
       <widget class="QTableWidget" name="userIdTableWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>81</width>
          <height>216</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>사용자 아이디</string>
         </property>
        </column>
       </widget>
       <widget class="QTableWidget" name="userAccountsTabelWidget">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>0</y>
          <width>281</width>
          <height>181</height>
         </rect>
        </property>
        <property name="verticalScrollMode">
         <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
        </property>
        <property name="horizontalScrollMode">
         <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
        </property>
        <column>
         <property name="text">
          <string>계좌 종류</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>잔액</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>기간</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>계좌 생성일</string>
         </property>
        </column>
       </widget>
       <widget class="QPushButton" name="userDeleteButton">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>190</y>
          <width>100</width>
          <height>30</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>유저 삭제</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="logoutBtn">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>로그 아웃</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="withdrawBtn">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>회원 탈퇴</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
